files/zh-cn/web/api/xmldocument/async/index.html

```diff
@@ -5,15 +5,15 @@
 ---
 <p>{{APIRef("DOM")}}{{Non-standard_header}}{{Deprecated_header}}</p>
 
-<p><code>document.async</code> 可以被赋值,用来表明 <code><a href="/zh-cn/DOM/document.load" title="zh-cn/DOM/document.load">document.load</a></code> 被调用时,是使用异步模式还是同步模式进行请求. <code>true</code> 为默认值, 表明该文档应该被异步加载.</p>
+<p><code>document.async</code> 可以被赋值，用来表明 <code><a href="/zh-cn/DOM/document.load" title="zh-cn/DOM/document.load">document.load</a></code> 被调用时，是使用异步模式还是同步模式进行请求。<code>true</code> 为默认值，表明该文档应该被异步加载。</p>
 
-<p>(从Gecko 1.4alpha 开始,文档请求可以以异步模式进行,在这之前,只有同步模式可用.)</p>
+<p>(从 Gecko 1.4alpha 开始，文档请求可以以异步模式进行，在这之前，只有同步模式可用.)</p>
```

Comment:
seems not take effect:

```suggestion
<p>(从 Gecko 1.4alpha 开始，文档请求可以以异步模式进行，在这之前，只有同步模式可用。)</p>
```